authors/apps/authentication/tests.py

```diff
@@ -0,0 +1,139 @@
+import json
```

Comment:
Good work, A quick update though, please put the `tests.py` file in a folder labeled `tests` and then rename the file to `test_api.py`